cartoframes/assets/src/bundle.js

```diff
@@ -0,0 +1,501 @@
+var init = (function () {
```

Comment:
If this is the resultant bundle, should this file be commited in an src folder?

Reply:
I've been discussing this with @rjimenezda. First of all, I think yes, it has to be in the folder (although we can place it in a `/dist` directory). Since this is the Python package, we need to have it there to be loaded. Another option would be to have the scripts in a different repository, but that'd be IMO hard to maintain. Another problem we've is that we may forget to commit the bundle file, and we're trying to find the best way to ensure this file has been updated if any javascript file has changed.

Reply:
Ok, thanks for the explanation. If the bundle has to be committed I wouldn't place in a `/dist` directory as it increases the chances we forget to do it.